### PR TITLE
Add signature file

### PR DESCRIPTION
**Pull request by dlandon**
Add signature file for ownCloud 10.

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<namespace>Tasks</namespace>
 	<category>productivity</category>
 	<licence>AGPL</licence>
-	<author>Raimund Schlüßler</author>
+	<author>Dan Landon</author>
 	<description>Task App for Calender TODOs</description>
 	<ocsid>164356</ocsid>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: you can remove ocsid

File: appinfo/signature.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
+{
```
Comment (DeepDiver1975) [inline appinfo/signature.json]: the signature is usually not checked into git

File: appinfo/info.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
 	<namespace>Tasks</namespace>
 	<category>productivity</category>
 	<licence>AGPL</licence>
-	<author>Raimund Schlüßler</author>
+	<author>Dan Landon</author>
 	<description>Task App for Calender TODOs</description>
```
Comment (DeepDiver1975) [inline appinfo/info.xml]: in addition to the description (which is the long description) you need to add a summary tag - being the short description.

Comment (DeepDiver1975) [reply appinfo/info.xml]: furthermore at least one screenshot is required

File: appinfo/signature.json (hunk 2 above)
Comment (dlandon) [reply appinfo/signature.json]: Where do I add a screen shot?   What format .jpg?  Summary tag is <summary></summary>?  You mean the signature.json file is not checked into github?

Comment (DeepDiver1975) [reply appinfo/signature.json]: example: https://github.com/owncloud/calendar/blob/master/appinfo/info.xml#L22

you can add the screenshot to this repo and link to it in the xml

Comment (DeepDiver1975) [reply appinfo/signature.json]: > ? You mean the signature.json file is not checked into github?

yes

**Pull request merged**